travis-ci/managers/fuzzit.sh

```diff
@@ -48,10 +48,30 @@ FUZZIT_ARGS="--type ${FUZZING_TYPE} --branch ${FUZZIT_BRANCH} --revision ${TRAVI
 wget -O fuzzit https://github.com/fuzzitdev/fuzzit/releases/latest/download/fuzzit_Linux_x86_64
 chmod +x fuzzit
 
-find out/ -maxdepth 1 -name 'fuzz-*' -executable -type f -exec basename '{}' \; | xargs --verbose -n1 -I%FUZZER% ./fuzzit create job ${FUZZIT_ARGS} %FUZZER%-asan-ubsan out/%FUZZER% ${FUZZIT_ADDITIONAL_FILES}
+# Simple wrapper which retries given command up to three times if it fails
+_retry() {
+    local EC=1
+
+    for _ in {0..2}; do
+        if "$@"; then
+            EC=0
+            break
+        fi
+
+        sleep 1
+    done
+
+    return $EC
+}
+
+while read -r fuzzer; do
+    _retry ./fuzzit create job ${FUZZIT_ARGS} ${fuzzer}-asan-ubsan out/${fuzzer} ${FUZZIT_ADDITIONAL_FILES}
+done <<< "$(find out/ -maxdepth 1 -name 'fuzz-*' -executable -type f -exec basename '{}' \;)"
```

Comment:
I can't seem to wrap my head around this :-) Could you replace it with
```sh
find ... | while read -r fuzzer; do
    ...
done
```
?

FWIW if it keeps failing I think it would make sense to consider turning it off and maybe eventually switching to https://google.github.io/oss-fuzz/getting-started/continuous-integration/.

Reply:
> I can't seem to wrap my head around this :-) Could you replace it with
> 
> ```shell
> find ... | while read -r fuzzer; do
>     ...
> done
> ```
> 
> ?

Hah, gladly. Not sure why this *monstrosity* was the first thing that came to my mind :-)
> 
> FWIW if it keeps failing I think it would make sense to consider turning it off and maybe eventually switching to https://google.github.io/oss-fuzz/getting-started/continuous-integration/.

Never heard of CIFuzz before, but it sounds really great (and I'd finally get an excuse to try out GH workflows). Will definitely look into that ~tomorrow~ today, thanks!